specification/cognitiveservices/data-plane/CustomSearch/v1.0/CustomSearch.json

```diff
@@ -0,0 +1,598 @@
+				"$ref": "#/definitions/Identifiable"
+			}],
+			"type": "object",
+			"properties": {
+				"webSearchUrl": {
+					"description": "The URL To Bing's search result for this item.",
+					"readOnly": true,
+					"type": "string"
+				}
+			}
+		},
+		"QueryContext": {
+			"description": "Defines the query context that Bing used for the request.",
+			"type": "object",
+			"required": [
+				"originalQuery"
+			],
+			"properties": {
+				"originalQuery": {
+					"description": "The query string as specified in the request.",
+					"type": "string"
+				},
+				"alteredQuery": {
+					"description": "The query string used by Bing to perform the query. Bing uses the altered query string if the original query string contained spelling mistakes. For example, if the query string is \"saling downwind\", the altered query string will be \"sailing downwind\". This field is included only if the original query string contains a spelling mistake.",
+					"readOnly": true,
+					"type": "string"
+				},
+				"alterationOverrideQuery": {
+					"description": "The query string to use to force Bing to use the original string. For example, if the query string is \"saling downwind\", the override query string will be \"+saling downwind\". Remember to encode the query string which results in \"%2Bsaling+downwind\". This field is included only if the original query string contains a spelling mistake.",
+					"readOnly": true,
+					"type": "string"
+				},
+				"adultIntent": {
+					"description": "A Boolean value that indicates whether the specified query has adult intent. The value is true if the query has adult intent; otherwise, false.",
+					"readOnly": true,
+					"type": "boolean"
+				}
+			}
+		},
+		"WebWebAnswer": {
+			"description": "Defines a list of relevant webpage links.",
+			"x-ms-discriminator-value": "Web/WebAnswer",
+			"allOf": [{
+				"$ref": "#/definitions/SearchResultsAnswer"
+			}],
+			"type": "object",
+			"required": [
+				"value"
+			],
+			"properties": {
+				"value": {
+					"description": "A list of webpages that are relevant to the query.",
+					"type": "array",
+					"items": {
+						"$ref": "#/definitions/WebPage"
+					}
+				},
+				"someResultsRemoved": {
+					"description": "A Boolean value that indicates whether the response excluded some results from the answer. If Bing excluded some results, the value is true.",
+					"readOnly": true,
+					"type": "boolean"
+				}
+			}
+		},
+		"SearchResultsAnswer": {
+			"allOf": [{
+				"$ref": "#/definitions/Answer"
+			}],
+			"type": "object",
+			"properties": {
+				"queryContext": {
+					"$ref": "#/definitions/QueryContext",
+					"readOnly": true
+				},
+				"totalEstimatedMatches": {
+					"description": "The estimated number of webpages that are relevant to the query. Use this number along with the count and offset query parameters to page the results.",
+					"readOnly": true,
+					"type": "integer",
+					"format": "int64"
+				},
+				"isFamilyFriendly": {
```

Comment:
>isFamilyFriendly [](start = 5, length = 16)

Please provide a description

Reply:
Not stated in documentation. This is an indirect reference. There is no description for it in websearch spec

Reply:
Usually we don't have description for easily understood schema fields

 THis is same as below
https://github.com/Azure/azure-rest-api-specs/blob/current/specification/cognitiveservices/data-plane/WebSearch/v1.0/WebSearch.json